cmd/analyze/analyze.go

```diff
@@ -44,24 +47,55 @@ var AnalyzeCmd = &cobra.Command{
 	provide you with a list of issues that need to be resolved`,
 	Run: func(cmd *cobra.Command, args []string) {
 
+		if alert != "" {
+			explain = true
```

Comment:
What if I dont want AI or alerts?

Reply:
This mode is necessarily associated with AI since the router and the evaluator agent are calls to the AI.